app/views/api/backend_usages/edit.html.slim

```diff
@@ -1,12 +1,11 @@
-= content_for :sublayout_title, 'Edit Backend usage'
+= content_for :sublayout_title, 'Edit Backend Usage'
 
 = semantic_form_for @backend_api_config, url: admin_service_backend_usage_path(@service, @backend_api_config) do |form|
   = form.inputs do
     li
       = form.label :backend_api_id, 'Backend'
       = form.text_field :backend_api_id, class: 'text', value: @backend_api_config.backend_api.name, disabled: true
-    = form.input :path, hint: 'Changing this path will only affect the behaviour of this Backend within the context of this Product'
+    = form.input :path, hint: 'Changing the public path will only affect this product', label: 'Public Path'
```

Comment:
```suggestion
    = form.input :path, hint: 'Changing the public path will only affect this Product', label: 'Public Path'
```

Reply:
I did it on purpose as I understand that Docs team doesn't want us to use capitals like that. Not sure, @cntlsn do you know?